post-processing/post_processing.py

```diff
@@ -50,6 +51,10 @@ def run_post_processing(self, config: ConfigHandler):
         # FIXME (#issue #255): have an option to put this into a file (-s / --save flag?)
         print("Selected dataframe:")
         print(self.df[self.mask][config.plot_columns])
+        if self.debug:
+            print("CSV dataframe:")
+            print(self.df[self.mask][config.plot_columns + config.extra_columns])
+        self.df[self.mask][config.plot_columns + config.extra_columns].to_csv(str(self.log_path)+'/output.csv', index=True)  # Set index=False to exclude the DataFrame index from the CSV

```

Comment:
I did a quick investigation into why the tests are failing: you're trying to concatenate `self.log_path` with `"output.csv"`, but that won't always work because the `log_path` isn't necessarily a directory, it can also be a file. I think the best way to fix this is that we can either try to find an appropriate parent folder in the path, or let the user specify their own `output_path` where they'd like to save the CSV.

Reply:
As I said above, I prefer the former option :)